### ISSUE TITLE
Uncaught Error: not opened

### ISSUE THREAD

**Issue by Hihaj** [wallabyjs/atom-wallaby#39]
I'm getting this error in atom-wallaby quite often recently. Don't know exactly how to reproduce; sometimes it seems like it's triggered after the computer's been to sleep but sometimes I get it while working without interruption. Any ideas?

**Atom Version**: 1.8.0
**System**: Mac OS X 10.11.5
**Thrown From**: [atom-wallaby](https://github.com/wallabyjs/atom-wallaby) package, v1.0.8

### Stack Trace

Uncaught Error: not opened

```
At /Users/hihaj/.atom/packages/atom-wallaby/wallaby/node_modules/ws/lib/WebSocket.js:181

Error: not opened
    at WebSocket.send (/Users/hihaj/.atom/packages/atom-wallaby/wallaby/node_modules/ws/lib/WebSocket.js:181:16)
    at EventEmitter._.extend._send (/Users/hihaj/.atom/packages/atom-wallaby/wallaby/client.js:14:3889)
    at EventEmitter._.extend.fileClosedInEditor (/Users/hihaj/.atom/packages/atom-wallaby/wallaby/client.js:14:2860)
    at Object.n.dispose (/Users/hihaj/.atom/packages/atom-wallaby/wallaby-atom/index.js:14:16478)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:25:29
    at Set.forEach (native)
    at CompositeDisposable.module.exports.CompositeDisposable.dispose (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:24:26)
    at TextEditor.module.exports.TextEditor.destroyed (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:283:24)
    at TextEditor.module.exports.Model.destroy (/Applications/Atom.app/Contents/Resources/app.asar/src/model.js:31:58)
    at Pane.module.exports.Pane.destroyItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:639:18)
    at atom-tabs.TabBarView.closeTab (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/tabs/lib/tab-bar-view.js:321:26)
    at atom-tabs.TabBarView.closeAllTabs (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/tabs/lib/tab-bar-view.js:452:28)
    at Object.addElementCommands.tabs:close-all-tabs (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/tabs/lib/tab-bar-view.js:120:26)
    at atom-tabs.commandsWithPropagationStopped.(anonymous function) (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/tabs/lib/tab-bar-view.js:86:36)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:160:19)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:1106:28)
    at EventEmitter.outerCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/application-delegate.js:340:25)
    at emitThree (events.js:97:13)
    at EventEmitter.emit (events.js:175:7)
```

### Commands

```
  2x -0:07.2.0 tabs:close-all-tabs (div.title)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "language-c",
      "language-clojure",
      "language-go",
      "language-java",
      "language-make",
      "language-objective-c",
      "language-perl",
      "language-php",
      "language-python",
      "language-ruby",
      "language-ruby-on-rails",
      "language-toml",
      "metrics",
      "open-on-github",
      "language-mustache",
      "spell-check",
      "jshint"
    ],
    "openEmptyEditorOnStart": false,
    "themes": [
      "atom-dark-ui",
      "monokai"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.29.9
atom-wallaby, v1.0.8
language-csharp, v0.12.1
linter, v1.11.14
linter-csslint, v1.3.2
linter-eslint, v7.2.4
linter-scss-lint, v3.0.4
monokai, v0.18.0
react, v0.15.0

# Dev
No dev packages
```

**Comment by ArtemGovorov** [comment]
Thanks for the report. The issue is fixed and the fix is now published. You may wait for a few minutes before it updates automatically, or force the update by closing the editor, opening the `user/.atom/packages/atom-wallaby/` folder and deleting the `wallaby-atom` subfolder.

**Comment by Hihaj** [comment]
Wow, that was fast! Thanks!
mån 4 juli 2016 kl. 10:04 skrev Artem Govorov <notifications@github.com>:

**Comment by Hihaj** [comment]
@ArtemGovorov Are you sure the fix has been published? When I look at the releases for atom-wallaby it says that 1.0.8 is the latest version, published about a month ago?

**Comment by ArtemGovorov** [comment]
@Hihaj Yes, the fix is published. The `atom-wallaby` package is [just a loader/updater of wallaby.js Plugin and Core](https://wallabyjs.com/docs/intro/install.html#atom), it gets updated quite infrequently and does it the same way as all other Atom packages that you may have installed. Wallaby Plugin for Atom (the one that was fixed) gets updated automatically, you may see its version in the Dev Tools console in Atom along with the Wallaby.js core version. The version with the fix in 1.0.28, if it's not, you may follow the instructions [I have listed above](https://github.com/wallabyjs/atom-wallaby/issues/39#issuecomment-230228247) to force the update.
